app/models/ability.rb

```diff
@@ -45,16 +45,17 @@ def not_signed_in
     # can view Commercials of confirmed Events
     can :show, Commercial, commercialable_type: 'Event', commercialable_id: Event.where(state: 'confirmed').pluck(:id)
     can [:show, :create], User
+
+    can [:new, :create], Registration do |registration|
```

Comment:
That implies that a not_signed_in user can create a Registration even if ichain is enabled. That's not a valid ability.